go.mod

```diff
@@ -1,32 +1,30 @@
 module github.com/99designs/gqlgen
 
-go 1.12
+go 1.13
 
 require (
-	github.com/agnivade/levenshtein v1.0.3 // indirect
-	github.com/go-chi/chi v3.3.2+incompatible
-	github.com/gogo/protobuf v1.0.0 // indirect
-	github.com/gorilla/context v0.0.0-20160226214623-1ea25387ff6f // indirect
-	github.com/gorilla/mux v1.6.1 // indirect
-	github.com/gorilla/websocket v1.2.0
-	github.com/hashicorp/golang-lru v0.5.0
+	github.com/go-chi/chi v4.0.3+incompatible
```

Comment:
Theres too much changing in here. Can you revert these mod/sum changes and just update urfave via
```
go get github.com/urfave/cli/v2
go mod tidy
```

Reply:
Done!